portfolio/src/main/webapp/script.js

```diff
@@ -28,10 +28,33 @@ function addRandomGreeting() {
 }
 
 function addRandomNameGreeting(){
-    console.log("Fetching a randome greeting with a name   ");
+    console.log("Fetching a random greeting with a name   ");
     fetch("/data").then(response => response.text()).then((name) => {
         document.getElementById('name-container').innerText = name;
     });
 
 }
 
+function postComments() {
+    fetch('/data').then(response => response.json()).then((comments) => {
+      const commentsListElement = document.getElementById('comments-container');
```

Comment:
can probably move getElementById out of the lambda as it's basically a constant.  better to have short lambda esp if they're executed more than once.